open-source.adoc

```diff
@@ -206,83 +206,81 @@ speaking, you should donate to me based on the utility you find my
-anyone who works in Bitcoin or holds significant amounts of the
+Notice that these points don't just hold true for Satoshi Nakamoto, but for
+anyone working in Bitcoin or holding significant amounts of the
 currency, to varying degrees.
 
 [[selectioncryptography]]
 === Selection cryptography
 
-Open source developers often use open source libraries developed by
-other people. This is a natural, and awesome part of a healthy
-ecosystem. But Bitcoin software deals with real money, and as such,
-developers need to be extra careful when choosing which third party
-libraries the software should depend on.
-
-In a
-https://btctranscripts.com/greg-maxwell/2015-04-29-gmaxwell-bitcoin-selection-cryptography/[philosophical
-talk about cryptography] (search for the part that starts with "`The
-art of selection cryptography`" or watch the
-https://youtu.be/Gs9lJTRZCDc?t=2236[video]), Gregory Maxwell wants to
-redefine the term cryptography which he believes is too narrow. He
-speaks about how, fundamentally, _information wants to be free_, and
+Open source developers often make use of open source libraries developed by
+other people. This is a natural and awesome part of any healthy
+ecosystem. But Bitcoin software deals with real money, and as such it needs to be extra careful when choosing which third party libraries it should depend on.
+
+In a philosophical
+https://btctranscripts.com/greg-maxwell/2015-04-29-gmaxwell-bitcoin-selection-cryptography/[talk
+about cryptography] (you may find the video
+https://youtu.be/Gs9lJTRZCDc?t=2236[here]), Gregory Maxwell wants to
+redefine the term "`cryptography`" which he believes to be too narrow. He
+explains that fundamentally _information wants to be free_, and
 makes his definition of cryptography based on that:
 
-[quote, Gregory Maxwell, The Art Of Selection Cryptography]
+[quote, Gregory Maxwell, Bitcoin Selection Cryptography (2015)]
 ____
 *Cryptography* is the art and science we use to fight the fundamental
 nature of information, to bend it to our political and moral will, and
 to direct it to human ends against all chance and efforts to
 oppose it.
 ____
 
-He then introduces the term _selection cryptography_, which is the art
-of selecting cryptographic tools, and explains why that is an
+He then introduces the term _selection cryptography_, referred to as the art
+of selecting cryptographic tools, and explains why it is an
 important part of cryptography. It revolves around how to select
-cryptographic libraries, tools, and practices, or as he says "`The
+cryptographic libraries, tools, and practices, or as he says "`the
 cryptosystem of picking cryptosystems`".
 
 Using concrete examples, he shows how selection cryptography can
-easily go horribly wrong and also proposes a list of things to think
-about to do better. A distilled version of that list is:
+easily go horribly wrong, and also proposes a list of questions
+you should always ask yourself when resorting to it.
```

Comment:
"Should always" is a bit too harsh. Suggested edit:

"you should always ask" -> "you could ask"